main.cpp

```diff
@@ -52,85 +52,85 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        numberSelector
+//  action 1:    enumerates even numbers
+numberSelector.enumerateEven();
+//  action 2:    checks 0 presence
+numberSelector.checkZeroPresence();
+//  action 3:    applies an inversion on an array
+numberSelector.invertArray();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        oscillator
+//  action 1:    produces sine
+oscillator.produceSine();
+//  action 2:    adds a limit threshold
+oscillator.setLimitThreshold();
+//  action 3:    modulates sine with Square
+oscillator.modulateSineWith();
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        appWindow
+//  action 1:    shows data analysed
+appWindow.showDataAnalysed();
+//  action 2:    shrinks a sub-module inside application
+appWindow.shrinkSubModule();
+//  action 3:    returns calculated free area of application
+appWindow.getFreeArea();
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        cube
+//  action 1:    returns the coordinates of all vortices as 2-dimension array
+cube.getVortices();
+//  action 2:    applies the rotation for an angle passed as an argument
+cube.rotateBy(angle);
+//  action 3:    applies the scale passed as an argument
+cube.scaleBy(scaleFactor);
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        searchLine
+//  action 1:    considering case sensitivity
+searchLine.considerCaseSensitivity();
+//  action 2:    filters result by date of last updates
+searchLine.showWithLatestUpdates();
+//  action 3:    clears it's content
+searchLine.clear();
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        appLauncher
+//  action 1:    checks for updates
+appLauncher.checkForUpdates();
+//  action 2:    hides ads
+appLauncher.hideAds();
+//  action 3:    toggles launch with windows start up
+appLauncher.setLaunchAtWindowsStartup();
 //  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        cat
+//  action 1:    runs in panic
+cat.runInPanic();
+//  action 2:    falls to sleep
+cat.sleep();
+//  action 3:    starts to purr
+cat.Purr();

```

Comment:
I'm marking this because:
1. your plain-english description is not an action, but is an ability or describes a state OR
2. the plain-english description is fine but the function name doesn't match OR
3. The way you wrote the name doesn't conform to the course coding standard:
    use `camelCase` for variable and function names.
    Type Names begin with a Capitalized first letter. All other symbols begin with a lower-case first letter.
- [ ] revise